src/client/features/search/landing-box.js

```diff
@@ -47,9 +47,18 @@ const pcFallback = (unrecognized,genes) => {
 
 const idToLinkConverter = (ids)  =>{
   const dbSet = databases.filter(databaseValue => ids[databaseValue.database]);
-  return _.assign({},
+  let dbs =  _.assign({},
     ...dbSet.map(database=>({[database.database]:database.url+database.search+ids[database.database].replace(/[^a-zA-Z0-9:]/g)}))
   );
+
+  let links = [];
+  usedDatabases.forEach((usedDatabase)=>{
+    let dbLink = dbs[usedDatabase.configName];
+    if (dbLink != null) {
+      links.push({"link":dbLink,"displayName":usedDatabase.displayName});
```

Comment:
1. This is a minor nitpick but to keep the coding style consistent: for object keys you generally don't need to use strings you can just do something like this.
``` js
links.push({ link: dbLink, displayName: usedDatabase.displayName });
```

Reply:
The only place where quotation marks are mandatory on keys is in JSON files.